### PR TITLE
🚑 use preact instead of react to reduce bundle size

### PR DESCRIPTION
**Pull request by healthpackdev**
You can use preact.

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ const withImages = require("next-images");
 
 module.exports = withImages({
 	reactStrictMode: true,
+	webpack(config) {
+		// replace react with preact to reduce bundle size and faster DevServer
+		Object.assign(config.resolve.alias, {
+			"react-dom": "preact/compat",
+			react: "preact/compat",
+		});
```
Comment (barbarbar338) [inline next.config.js:11]: you have to return a webpack config in this function. If you fix that i can merge :)

File: package.json [hunk 2]
```diff
@@ -1,71 +1,72 @@
...
+		"autoprefixer": "^10.2.6",
+		"css-loader": "^5.2.6",
+		"eslint": "^7.29.0",
+		"eslint-config-next": "^11.0.1",
+		"eslint-plugin-react": "^7.24.0",
+		"eslint-plugin-react-hooks": "^4.2.0",
+		"onchange": "^7.1.0",
+		"postcss": "^8.3.5",
+		"prettier": "^2.3.2",
+		"react": "^17.0.2",
```
Comment (barbarbar338) [inline package.json:54]: also we can remove react if we want to use preact

**Review by barbarbar338 — CHANGES_REQUESTED** [review]
Needs a litte job to done

File: package.json (hunk 2 above)
Comment (healthpackdev) [reply package.json:54]: I've moved it to devDependencies in case it might throw type errors or warnings

File: next.config.js (hunk 1 above)
Comment (healthpackdev) [reply next.config.js:11]: sorry! I added.

File: package.json (hunk 2 above)
Comment (barbarbar338) [reply package.json:54]: Well then, let's not delete it for now.

**Comment by barbarbar338** [comment]
Thanks for your awesome contribution 💪✨

**Comment by healthpackdev** [comment]
Also if you want to use React Devtools. You can install preact-devtools to your browser and then
```js
if(CONFIG.DEV) {
 require('preact/debug');
}
``` 
add this line in to _app.tsx

**Pull request merged**